packages/airnode-node/src/providers/actions.test.ts

```diff
@@ -1,3 +1,6 @@
+// Workaround due to issue https://api3dao.atlassian.net/browse/AN-443
+// Can be removed once the issue is resolved
+console.log();
```

Comment:
You can remove it now - the tests will pass. This is only an issue when logger is enabled (not silenced) or when using console.log